websites/B/BeatLeader/metadata.json

```
@@ -13,7 +13,7 @@
 		"replay.beatleader.xyz",
 		"royale.beatleader.xyz"
 	],
-	"version": "1.1.1",
+	"version": "1.2.0",
 	"logo": "https://i.imgur.com/TWPjaYR.png",
```

While you're at it, lets change to logo to 512x512
```suggestion
	"logo": "https://i.imgur.com/b5HCP1a.png",
```